dev/EVAndPATH/Microsoft.Process.Environment/module.g.cpp

```diff
@@ -0,0 +1,61 @@
+// WARNING: Please don't edit this file. It was generated by C++/WinRT v2.0.210122.3
+
+#include "pch.h"
+#include "winrt/base.h"
+void* winrt_make_Microsoft_Process_Environment_EnvironmentManager();
+
+bool __stdcall winrt_can_unload_now() noexcept
```

Comment:
Looks like dev/EVAndPATH is making its own DLL. Why? Shouldn't it go into the Microsoft.ProjectReunion.dll?

If so, you should have a SharedProject with your product code, add a Reference to your SharedProject in the ProjectReunion_DLL project and any 'module level changes' need to be coordinated through that. AppLifecycle did this to a degree and the [Dynamic Depedencies PR](https://github.com/microsoft/ProjectReunion/pull/393) goes much further (important additions like a DllMain and invoking coordinated code from multiple SharedProjects).

I thought there was a README about the SharedProject game but I can't seem to find it. Did someone change or remove it in the past few months?

Reply:
The SharedProject documentation was removed as it was a demo prior to real code coming in.  Definitely agree though - please put your code into the shared project structure:

* Code goes into dev\Environment (better to use fuller names)
* Test goes into test\Environment
* Add a *.vcxitems (see dev\AppLifecycle\AppLifecycle.vcxitems)

Reply:
Sorry.  I am confused.  there is no dev/environment.  Are you talking about putting the project directly under dev, or are you suggesting to put the project under dev/Microsoft.Process.Environment?

Reply:
@DrusTheAxe Moved my code over to a shared project and ProjectReunion_DLL references this as a shared project.  Is this what you are looking for?